cloud/linode/instances.go

```diff
@@ -13,12 +14,46 @@ import (
 	cloudprovider "k8s.io/cloud-provider"
 )
 
+type nodeCache struct {
+	sync.RWMutex
+	nodes      map[int]*linodego.Instance
+	lastUpdate time.Time
+	ttl        time.Duration
```

Comment:
should this be a flag we can change if we need to?

Reply:
Yeah, could be. It's just that the init for this (and all its parent inits) is not via main.go, it goes through `init()`, so it'd have to be an env variable.

Reply:
I think it might be worth doing just in the off chance we need to tune this, then it won't be a pain. We also might want to have different ttl for the child clusters vs the CPC.